src/wechaty/user/url_link.py

```diff
@@ -44,7 +44,7 @@ def create(
             title = title[0] if len(title) else url
         if thumbnail_url is None:
             thumbnail_url = html.xpath('//meta[@property="og:image"]/@content')
-            thumbnailUrl = thumbnail_url[0] if len(thumbnail_url) else ""
+            thumbnail_url = thumbnail_url[0] if len(thumbnail_url) else ""
         if description is None:
```

Comment:
As commented above, there is more consice checking style:

```python
if not thumbnail_url:
    pass
```

Please change them in project. Thanks a lot.